ecs-cluster/terraform.tf

```diff
@@ -1,8 +1,3 @@
-terraform {
```

Comment:
where are we storing tfstate?

Reply:
It's handled in the private `-deploy` repo. `backend "s3" {}` on it's own is useless, since there's no bucket configured.

Reply:
Turns out you were right :-(